lib/modules/betteReddit.js

```diff
@@ -219,8 +219,12 @@ module.go = () => {
 function commentsLinksNewTabs(ele = document.body) {
 	const links = ele.querySelectorAll('.thing div.md a');
 	links::forEachChunked(link => {
-		link.target = '_blank';
-		link.rel = 'noopener noreferer';
+		const { baseURI, href } = link;
+
+		if (!(href.includes(baseURI) && (href.includes('#!settings') || href.includes('#res:settings')))) {
```

Comment:
I was thinking a link eg to https://www.reddit.com/domain/siteIWantToFilter.com#res:settings/filteReddit/domain, but don't worry about it -- I'd rather get this in and working well for the general case.